.drone.yml

```diff
@@ -16,54 +16,43 @@ steps:
     - pull_request
     - tag
 
-- name: build
-  image: golang:1.16
-  volumes:
-  - name: dockersock
-    path: /var/run
-  commands:
-  - sleep 20
-  - TAG=${DRONE_TAG} make
+- name: docker-build
+  image: plugins/docker
+  settings:
+    dockerfile: Dockerfile
+    repo: "rancher/gke-operator"
+    tag: "${DRONE_TAG}"
+    dry_run: true
   when:
     event:
-    - tag
+    - pull_request
```

Comment:
Hmm, maybe you don't need docker build to run for docker-publish step to work. Looks like other drone files don't do that either.

Reply:
Looks like build failed cause it ran without tag specified. You can set it directly with param.

Reply:
> Hmm, maybe you don't need docker build to run for docker-publish step to work.

Right, it looks like docker-publish works independently

> Looks like build failed cause it ran without tag specified.

Oops. I don't see an example of `param` but this [istio example](https://github.com/rancher/istio-installer/blob/master/.drone.yml#L7-L14) hardcodes the tag for this step so maybe that will work?